### ISSUE TITLE
Change scrolling speed

### ISSUE THREAD

**Issue by TwentyXX** [warpdotdev/Warp#4178]
### Describe the solution you'd like?

I would like an option to freely change the scroll speed of the main terminal screen.

### Is your feature request related to a problem? Please describe.

The screen scrolling seemed a little slow. From what I saw, the option to change the scroll speed did not seem to exist yet, so I submitted a request.

### How important is this feature to you?

3

### Warp Internal (ignore) - linear-label:39cc6478-1249-4ee7-950b-c428edfeecd1

None

**Comment by dannyneira** [comment]
Thanks for this feature request @TwentyXX!

To anyone else interested in this feature, please add a 👍 to the original post at the top to signal that you want this feature, and subscribe if you'd like to be notified.

**Comment by gnusam** [comment]
Yes, adapt scroll speed or add a meta key to scroll slow of fast. Like scroll without key, you scroll slow and atl+scroll or ctrl+scoll = fastscroll :)

**Comment by heysweet** [comment]
Hello! It looks like we shipped `Settings > General > Lines scrolled by mouse wheel interval` back in March. Is this issue resolved for you @TwentyXX @gnusam ?
